desktop/release-notes.md

```diff
@@ -24,11 +24,63 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.21.0
+
+{% include release-date.html date="2023-06-29" %}
+
+{% include desktop-install.md all=true version="4.21.0" build_path="/" %}
+
+### New
+
+- Added support for new Wasm runtimes: slight, spin, and wasmtime.  Users can download Wasm runtimes on demand when the containerd image store is enabled.
+- Added Rust server support to Docker init.
+- New [Builds view](use-desktop/builds.md) that lets you inspect builds and manage builders.
+- New [Builds view](use-desktop/builds.md) (Beta) that lets you inspect builds and manage builders. This can be found in the **Features in Development** tab in Docker Desktop.
```

Comment:
```suggestion
- Beta release of the **Builds** view that lets you inspect builds and manage builders. This can be found in the **Features in Development** tab in **Settings**.
```

Follows a similar pattern we've followed before. I've also removed the link to the page (that isn't published yet) so we can get the PR merged. It's easier for me to retrospectively add links after merge

Reply:
Right! That makes sense! I've updated it accordingly :)